website/server/libs/slack.js

```diff
@@ -9,6 +9,10 @@ const SLACK_FLAGGING_URL = nconf.get('SLACK:FLAGGING_URL');
 const SLACK_FLAGGING_FOOTER_LINK = nconf.get('SLACK:FLAGGING_FOOTER_LINK');
 const SLACK_SUBSCRIPTIONS_URL = nconf.get('SLACK:SUBSCRIPTIONS_URL');
 const BASE_URL = nconf.get('BASE_URL');
+const IS_PRODUCTION = nconf.get('IS_PROD');
+
+const SKIP_FLAG_METHODS = IS_PRODUCTION && !SLACK_FLAGGING_URL;
+const SKIP_SUB_METHOD = IS_PRODUCTION && !SLACK_SUBSCRIPTIONS_URL;
```

Comment:
Just checking: This won't prevent the message being sent to slack when a post is flagged in staging, beta, gamma, and delta, will it? We do want flagging to work normally there, both for testing flagging and for cases where mods are using staging or the other sites for their normal use of Habitica (which is much of the time for staging). It might be worth adding a comment to clarify that.

Reply:
do staging / beta / gamma / delta have the "IS_PROD" value? if not I may have to rethink this skips

Reply:
I'm not sure. I think they would not. However if they DO have the `SLACK_FLAGGING_URL` and `SLACK_SUBSCRIPTIONS_URL` (which I expect they do since flagging works there now, at least for staging) then the `SKIP_....` constants will be false, which is what we want. 

Although I'm now wondering if the `SKIP_...` constants were meant to be false when IS_PROD is false. We wouldn't need the slack messages to be sent from flags made on a contributor's local install for instance. 
Should it be more like this:
`SKIP_FLAG_METHODS = !SLACK_FLAGGING_URL || !IS_PRODUCTION || !IS_STAGING || ....[beta, gamma, delta]`;
We'd want to tidy that up I guess, and I might be on the wrong track anyway. :)

Reply:
@paglias @SabreCat do we have any other config-/env-vars to identify if we are running under staging / beta / etc ? couldn't find any checks in our current code

Reply:
We don't have a separate env var but staging runs in production